rs/consensus/src/consensus/validator.rs

```diff
@@ -1095,17 +1095,24 @@ impl Validator {
                 // Ensure the proposal has a different hash from the validated
                 // block of same rank. Then we can construct the proof.
                 if proposal.content.get_hash().get_ref() != existing_metadata.content.hash() {
+                    let proof = EquivocationProof {
+                        signer: proposal.signature.signer,
+                        version: proposal.content.version().clone(),
+                        height: proposal.height(),
+                        subnet_id: self.replica_config.subnet_id,
+                        hash1: proposal.content.get_hash().clone(),
+                        signature1: proposal.signature.signature.clone(),
+                        hash2: CryptoHashOf::new(existing_metadata.content.hash().clone()),
+                        signature2: existing_metadata.signature.signature,
+                    };
+                    info!(
+                        self.log,
+                        "Equivocation found for node {:?}. Producing equivocation proof: {:?}",
+                        proof.signer,
```

Comment:
`signer` is part of `proof` already, so I guess it's not necessary to print it twice (unless it's more readable that way)?

```suggestion
                        self.log,
                        "Producing equivocation proof: {:?}",
```

Reply:
done